Code/GraphMol/Descriptors/Property.cpp

```diff
@@ -162,11 +147,23 @@ std::vector<std::string> Properties::getPropertyNames() const {
 std::vector<double> Properties::computeProperties(const RDKit::ROMol &mol) const {
   std::vector<double> res;
   res.reserve(m_properties.size());
-  BOOST_FOREACH(boost::shared_ptr<PropertyFxn> prop, m_properties) {
-    res.push_back(prop->compute(mol));
+  BOOST_FOREACH(boost::shared_ptr<PropertyFunctor> prop, m_properties) {
+    res.push_back((*prop)(mol));
   }
   return res;
 }
 
+void t() {
```

Comment:
I think this is leftover from some earlier testing? Doesn't seem to be used anywhere.